css-three-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+.row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container {
+  width: 80%;
+  margin: auto;
+  background-color: #1996da;
+  padding-right: 10rem;
+  padding-top: 3rem;
+  padding-left: 10rem;
+  padding-bottom: 2rem;
+}
+
+.column {
+  display: block;
+}
+
+h1 {
+  text-align: center;
+  color: white;
+  font-family: 'Lato', sans-serif;
+  margin: 3rem;
+}
+
+h3 {
+  color: white;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+}
+
+p {
+  color: #dcdddd;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+}
+
+.picture {
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  border-radius: 400px;
+  width: 35%;
+  position: relative;
+  left: 86px;
```

Comment:
What is the `left` doing here? Is this to center it? You might try `margin: auto`, I think it will give you the same effect without needing a `px` measured movement that would be hard to maintain after mobile responsiveness in the future.